tests/pages/components/github-issue-select.js

```diff
@@ -0,0 +1,37 @@
+import { attribute, findElementWithAssert, collection } from 'ember-cli-page-object';
+import { clickTrigger, nativeMouseUp } from 'code-corps-ember/tests/helpers/ember-power-select';
+
+export default {
+  scope: '.github-issue-select',
+
+  openDropdown() {
```

Comment:
I'll look into it, but I think there was a problem with that page object working for a power-select that isn't rendered in place.